fourkitchens/suhumsci
# SHS-4487: CKEditor style fixes

# READY FOR REVIEW

## Summary
- Change class for applying CKEditor styles, to ensure that the style plugin previews are correctly styled
- Fix styles of lists

## Steps to Test
1. Create a flexible page. Go to the Main Region Text area field
2. Confirm that the previews for the buttons in the "Styles" dropdown are correctly styled
3. Add some list content (both ordered and unordered). Save the content and confirm that the final styles match the styles in the editor


## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
